modules/product/src/main/java/com/opengamma/strata/product/fx/ResolvedFxVanillaOption.java

```diff
@@ -56,14 +52,6 @@
     implements ResolvedProduct, ImmutableBean, Serializable {
 
   /**
-   * Whether the option is put or call.
-   * <p>
-   * A call gives the owner the right, but not obligation, to exercise the underlying foreign exchange.
-   * A put gives a similar option to exercise the inverse of the underlying.
-   */
-  @PropertyDefinition(validate = "notNull")
-  private final PutCall putCall;
-  /**
    * Whether the option is long or short.
    * <p>
```

Comment:
Long indicates that the option owner has the right to enter into the transaction.
Short indicates that the decision on entering the option at expiry will be exercised by the other party.

Remove the existing long/short description which is incorrect.